bin/run-in-docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+
+# Synopsis:
+# Run the representer on a solution using the representer Docker image.
+# The representer Docker image is built automatically.
+
+# Usage:
+# ./bin/run-in-docker.sh two-fer ~/folder/to/solution/input ~/folder/to/representer/output
+
+# If any required arguments is missing, print the usage and exit
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+    echo "usage: ./bin/run-in-docker.sh exercise-slug path/to/solution/folder/ path/to/output/directory/"
+    exit 1
+fi
+
```

Comment:
Might I suggested adding a `set -e`:

```suggestion
set -e

```

We have that included in the latest version of the generic test runner, but many of the existing tooling repos don't yet have it. This has caused issues in the past, as you could have a failing `docker build` command but still have the `docker run` succeed as that would then download the latest version from Docker hub, not using the (broken) local image.

See https://github.com/exercism/generic-test-runner/blob/main/bin/run-in-docker.sh#L19-L20

Reply:
Oh I like that, done.